packages/mui-base/README.md

```diff
@@ -1,4 +1,9 @@
-# Base UI
+<!-- markdownlint-disable-next-line -->
+<p align="center">
+  <a href="https://mui.com/base-ui/" rel="noopener" target="_blank"><img width="150" height="133" src="https://mui.com/static/logo.svg" alt="Base UI logo"></a>
```

Comment:
@danilo-leal We will be able to change the logos, e.g. Base UI here.

https://github.com/oliviertassinari/material-ui/blob/docs-improve-readme/packages/mui-base/README.md